### ISSUE TITLE
PNPM Support

### ISSUE THREAD

**Issue by Camo30** [gsoft-inc/azure-devops-npm-auth#21]
Hi, thanks for this great tool!

It would be great if this would also support alternative package managers like [PNPM](https://pnpm.io/) even when NPM is not installed. Currently this does not work, when executing the preinstall script the following output appears:
```
'npm' is not recognized as an internal or external command,
operable program or batch file.
child_process.js:655
    throw err;
    ^

Error: Command failed: npm config get userconfig
operable program or batch file.
```

**Comment by yohanb** [comment]
Hey @Camo30 ! 
It would be a good amount of work to integrate alternative tools like PNPM.  PRs are welcome if you're motivated! :)

**Comment by Camo30** [comment]
Do you have an idea how to accomplish this? Maybe at first try with npm command, if it fails try with pnpm command...so some trial and error? Although this is not very elegant.

**Comment by Camo30** [comment]
@yohanb Thanks for merging! Could you bump the npm package to the latest version?

**Comment by yohanb** [comment]
@Camo30 done!  Published under version 1.0.13.
